Application/src/visuals/Renderable.java

```diff
@@ -20,4 +20,10 @@
 
     // Returns the Object's Coordinates.
     Coordinates getCoordinates();
+
+    // Returns the name for this object.
+    String getName();
```

Comment:
Whats the point of this line?

Reply:
ActorClass implements Renderable, therefore it needs to be defined there. It's then called in Draw.

Reply:
I missed that it was an interface my bad